app/models/payment.rb

```diff
@@ -49,7 +48,7 @@ class Payment < ApplicationRecord
   # order status 'checkout_complete'.  On the SHF side, that translates to a
   # completed payment ('paid') for the user fee (e.g. a membership fee).
   #
-  # NOTE: In May 2021, we switched the payment processor from HIPS to Klarna.
+  # NOTE: In October 2021, we switched the payment processor from HIPS to Klarna.
 
   ORDER_PAYMENT_STATUS = {
```

Comment:
Please add some comments that explain what you mean with the "HIPS" or "Klarna" comments on each line below.  (Picture someone looking at this code for the first time that may not have worked with any payment processors before.)
Short version: some of these statuses apply only to HIPS, some to Klarna.  Maybe there are some that apply to both.